product_docs/docs/ocl_connector/17/ocl_rel_notes/17.2.0.1_ocl_release_notes.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: "EDB OCL Connector 17.2.0.1 release notes"
+navTitle: Version 17.2.0.1
+---
+
+Released: 20 Nov 2024
```

Comment:
Let's see what Eric says first, but I believe this could be 
```suggestion
Released: 21 Nov 2024
```
If yes, we'll have to change in the index as well.